applications/luci-app-cloudflared/Makefile

```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: GPL-3.0-only
```

Comment:
the Luci uses Apache 2.0 licence. In the same time the GPL-3.0 has a lot of requirements so maybe you can use the Apache 2.0 here?
Anyway you have to specificity the `PKG_LICENSE` in the Makefile